### PR TITLE
WIKI-1452:Enable scrolling when clicking to anchor link

### PR DESCRIPTION
**Pull request by Jihed525**

### CODE REVIEW

File: wiki-webapp/src/main/webapp/javascript/eXo/wiki/UIWikiPortlet.js [hunk 1]
```diff
@@ -35,7 +35,6 @@ $(document).ready(function(){
     { 
       var href = $(this).attr('href');
       if (href.lastIndexOf("#", 0) == 0) {
-        e.preventDefault();
```
Comment (Jihed525) [inline wiki-webapp/src/main/webapp/javascript/eXo/wiki/UIWikiPortlet.js]: I have just removed the "e.preventDefault();" that disable scrolling to anchor link.

Comment (ahamdi) [reply wiki-webapp/src/main/webapp/javascript/eXo/wiki/UIWikiPortlet.js]: e.preventDefault(); was added for a purpose and the code above should do the scrolling.
 Could you check why this code is not working ?

File: wiki-webapp/src/main/webapp/javascript/eXo/wiki/UIWikiPortlet.js [hunk 2]
```diff
@@ -367,29 +366,29 @@ UIWikiPortlet.prototype.fixImageUrl = function(lastUpdated) {
...
-        }
-      }
-      var newQueryString = "?" + queryStringBuilder + "lastUpdate=" + lastUpdated; 
-      if (fragment) {
-        src = baseURL + newQueryString + "#" + fragment;
-      } else if (src.indexOf("#")) {
-        src = baseURL + newQueryString + "#";
-      } else {
-        src = baseURL + newQueryString;
+      var fragmentParser = src.split("#");
```
Comment (ahamdi) [inline wiki-webapp/src/main/webapp/javascript/eXo/wiki/UIWikiPortlet.js]: Please reset the changes on the code above if they are not part of the current PR

File: wiki-webapp/src/main/webapp/javascript/eXo/wiki/UIWikiPortlet.js (hunk 1 above)
Comment (Jihed525) [reply wiki-webapp/src/main/webapp/javascript/eXo/wiki/UIWikiPortlet.js]: This issue no longer exists, Therefor, this PR is no longer needed so I am closing it.

File: wiki-webapp/src/main/webapp/javascript/eXo/wiki/UIWikiPortlet.js (hunk 2 above)
Comment (Jihed525) [reply wiki-webapp/src/main/webapp/javascript/eXo/wiki/UIWikiPortlet.js]: This issue no longer exists, Therefor, this PR is no longer needed so I am closing it.

**Pull request closed without merging**